mobile/src/main/res/values-el/strings.xml

```diff
@@ -20,9 +20,6 @@
     <string name="drawer_close">Συρτάρι-κλειστό</string>
     <string name="drawer_open">Συρτάρι-ανοικτό</string>
     <string name="erorr_invalid_url">Παρακαλώ εισάγετε ενα σωστό URL της μορφής \'πρωτόκολλο://διακομιστής:πόρτα/\'</string>
-    <string name="erorr_no_wifi_mcast_permission">H άδεια CHANGE_WIFI_MULTICAST_STATE δεν εγκρίθηκε, η ανεύρεση του openHAB θα απενεργοποιηθεί!</string>
```

Comment:
did you intend to remove these strings in this PR?

Reply:
Yes, they have been removed from all other string files.